arches_lingo/src/arches_lingo/components/generic/ComponentManager/ComponentManager.vue

```diff
@@ -0,0 +1,150 @@
+<script setup lang="ts">
+import { computed, markRaw, provide, ref } from "vue";
+
+import { useRoute } from "vue-router";
+
+import Splitter from "primevue/splitter";
+import SplitterPanel from "primevue/splitterpanel";
+
+import ComponentEditor from "@/arches_lingo/components/generic/ComponentManager/components/ComponentEditor.vue";
+
+import {
+    CLOSED,
+    EDIT,
+    MAXIMIZED,
+    MINIMIZED,
+    NEW,
+    VIEW,
+} from "@/arches_lingo/constants.ts";
+
+import type { Component } from "vue";
+
+const props = defineProps<{
+    componentData: {
+        component: Component;
+        componentName: string;
+        sectionTitle: string;
+        graphSlug: string;
+        nodegroupAlias: string;
+    }[];
+}>();
+
+const route = useRoute();
+
+const processedComponentData = ref(
```

Comment:
Reports are stale when you click in the hierarchy, I guess we need to react to the route and increment a key somewhere. Seems to be a pre-existing issue, but it's more noticeable now that you can view a concept report. Opened #248